qcodes/instrument_drivers/cryogenic/CryogenicSMS120C.py

```diff
@@ -71,6 +71,7 @@ def __init__(self, name, address, coil_constant=0.113375, current_rating=105.84,
         idn = self.IDN.get()
         print(idn)
```

Comment:
This should be handled by the `connect_message` method of the instrument. Unless the IDN response is too non-standard.